cclient/BUILD

```diff
@@ -1,28 +1,28 @@
 cc_library(
     name = "api",
-    srcs = glob([
-        "*.c",
-        "**/*.c",
-        "**/*.h",
-    ]),
+    srcs = [
+        "iota_api.c",
+        "iota_api.h",
```

Comment:
Header in the sources ?

Reply:
https://docs.bazel.build/versions/master/be/c-cpp.html#cc_binary.srcs

Reply:
Ah, I got what you mean, thanks.